Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "plane-split"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
I'm afraid we can't just patch update here. `euclid` is in public API, and thus this becomes a breaking change